### ISSUE TITLE
Inconsistent Lane Segmentation Labels between Training data and Sample data

### ISSUE THREAD

**Issue by zoeveryday** [ApolloScapeAuto/dataset-api#22]
Take image /road03/Record001/171206_025755592_Camera_5.jpg for example.

![image](https://user-images.githubusercontent.com/47616652/57517703-062e9100-734a-11e9-8979-2beb8bf1bc36.png)

Left Image is from Training data label set. Mid Image is the original one. Right Image comes from the Sample data (lane_marking_examples.tar.gz).

The color of lanes are quite different... also the some features are also missing in the Sample data set. Does this mean the label data in Sample data set are just for demonstrations? And can not be used for training tasks.

Thanks for your reply!

**Comment by cardwing** [comment]
Same question.

**Comment by dingfuzhou** [comment]
Thanks for your interesting. 

First of all, the sample image is a sub-sample of the dataset, which has been included in the training set.  
The sample data should share the same label as the training data. The difference may be caused by different color encoded strategies.

**Comment by cardwing** [comment]
@dingfuzhou Thanks for your reply. ^_^

**Comment by zoeveryday** [comment]
So I guess I will stick to the color encoding strategy used in training data set. Thanks for the reply. Issue closed.
